### ISSUE TITLE
Option to bump package.json#version rather than git tag

### ISSUE THREAD

**Issue by jacktuck** [webpro/release-it#387]
Hi, I was trying to use `release-it` (`release-it@7.6.3`) to bump a packages version quicker  and it looks like a good fit. The problem i'm having is, I'm working with a monorepo, so I don't want to tag when i publish, and i don't want to publish based on the last git tag version. Not sure if this is possible atm, though.

So my latest git tag was 0.0.7, and with the following configuration, when i run `release-it patch`, it attempts to publish 0.0.8, but i want to publish `0.0.3` since my latest git tag has no relationship to this package's version (0.0.2). 

In other words, if my latest git tag is 0.0.7, and my current package version is 0.0.2, how do i make `release-it` publish 0.0.3 instead and ignore the git tag?

```
{
  "requireUpstream": false,
  "requireCleanWorkingDir": false,
  "src": {
    "commit": false,
    "tag": false,
    "push": false
  },
  "github": {
    "release": false
  }
}
```

**Comment by webpro** [comment]
Well, release-it is mainly git-based/focused. However, in theory, it would probably not be very hard to add some flag/toggle and use `package.json#version` instead, together with not git-tagging the repo during the release. Not sure whether this is a good idea, though.

Just out of curiosity, why does the repo's git tag not match the main `package.json#version`?

**Comment by jacktuck** [comment]
@webpro Because I have one repo consisting of multiple packages ([see lerna](https://github.com/lerna/lerna)) and each package has a distinct version number as they are published independently

**Comment by webpro** [comment]
Oh, are you using `release-it` from inside a sub/package directory of the monorepo?

**Comment by jacktuck** [comment]
@webpro Correct :)

**Comment by webpro** [comment]
@jacktuck, I've released `7.7.0-try.0`.

Make sure to set `src.tag` to `false` (or `--no-src.tag`), and the bump should be based on `package.json#version`.

It still gives a warning about non-matching Git tag, and probably an incorrect changelog, but the end result is no Git tag, but still a version bump in `package.json`.

Would you like to give this a shot on your mono repo? Is the correct `package.json` updated?

**Comment by jacktuck** [comment]
@webpro LGTM! 👍 Thanks so much for this one.

**Comment by webpro** [comment]
Thanks for the quick response, @jacktuck. It's a bit hacky, but impact on code level seems minimal, so I'll probably include it in v7.7.0 shortly.

**Comment by webpro** [comment]
@jacktuck This feature has actually never been released (other than in this "try" version).

However, I have just released [v9.0.0-beta.0](https://github.com/webpro/release-it/releases/tag/9.0.0-beta.0) and I was wondering whether you could give this a shot on your workload. I've also documented it a little bit: https://github.com/webpro/release-it#monorepos

Happy to hear your findingd and thoughts!

**Comment by gangwarily** [comment]
@webpro:

Is there a way to use this feature in a non-monorepo? I tried using `--nosrc.tag` in the CLI command and it continued to use the git tag version.

We were using a very old version of release-it and we used to be able to bump a major version by updating the version in package.json. We unfortunately cannot call `release-it major` since we run in a CI with a static bash script that calls one of the npm scripts.

**Comment by webpro** [comment]
@gangwarily, using `--no-src.tag` (mind the dash) indeed has no effect with regards to falling back to `version` from `package.json`. It does fall back in two cases:

* If not in the root of the Git repo.
* If the git tag isn't a valid semver tag.

I guess you do have a valid latest Git tag?

**Comment by gangwarily** [comment]
@webpro yup! We recently bumped from 1.0.11 to 2.0.0 manually in package.json and when release it was run, it updated package.json to 10.12 since the plugin defaults to running `release it patch`.
